core/evm.go

```diff
@@ -33,6 +33,9 @@ type ChainContext interface {
 
 	// GetHeader returns the hash corresponding to their hash.
 	GetHeader(common.Hash, uint64) *types.Header
+
+	// GetAncestorHash return the hash of ancestor at the given number
+	GetAncestorHash(child *types.Header, number uint64) common.Hash
```

Comment:
Do we still need the original `GetHeader`?

Reply:
Not in the interface, but we still need the implementation(s).. I can remove it from the interface